### PR TITLE
Window switching binding

### LINKED ISSUE #230: Implement some kind of alt-tab command for rootston
Just a general usability feature

### PR DESCRIPTION
**Pull request by versusvoid**
Fix #230

### CODE REVIEW

File: rootston/config.c [hunk 1]
```diff
@@ -298,15 +298,18 @@ struct roots_config *parse_args(int argc, char *argv[]) {
...
-		bc->modifiers = WLR_MODIFIER_LOGO;
-		bc->keysyms_len = 2;
-		bc->keysyms = calloc(1, bc->keysyms_len * sizeof(xkb_keysym_t));
-		bc->keysyms[0] = XKB_KEY_Meta_L;
-		bc->keysyms[1] = XKB_KEY_q;
+		const char default_config[] =
+			"[bindings]\n"
+			"Logo+Shift+e=exit\n"
+			"Ctrl+q=close\n"
+			"Alt+Tab=next window\n";
```
Comment (acrisci) [inline rootston/config.c]: Is it possible to make a function that adds the default bindings programatically that is shared with the parser?

File: rootston/keyboard.c [hunk 2]
```diff
@@ -32,6 +32,11 @@ static void keyboard_binding_execute(struct roots_keyboard *keyboard,
 		if (keyboard->input->last_active_view != NULL) {
 			view_close(keyboard->input->last_active_view);
 		}
+	} else if (strcmp(command, "next window") == 0) {
```
Comment (acrisci) [inline rootston/keyboard.c]: I know we don't have a parser, but `next_window` seems a bit nicer to me so we have one command per word.

Comment (versusvoid) [reply rootston/keyboard.c]: Oh_yeah,_totaly.

**Comment by acrisci** [comment]
👍 looks good

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**